modules/system-info/reporters/server.php

```diff
@@ -410,4 +383,77 @@ public function get_elementor_library() {
 			'value' => 'Connected',
 		];
 	}
+
+	/**
+	 * Check write permissions by provided paths.
+	 *
+	 * @since 3.7.0
+	 * @access public
+	 *
+	 * @param array $paths_to_check
+	 * @param array $errors
+	 *
+	 * @return array {
+	 *      @type string value: Writing permissions status.
+	 *      @type bool   warning: Whether to display a warning. True if some required.
+	 * }
+	 */
+	public function check_write_permissions( array $paths_to_check, $errors = [] ) {
+		$write_problems = $errors;
```

Comment:
No need for this renaming... You can just call the parameter `$write_problems`

Reply:
Done